src/components/__tests__/ListItem.spec.tsx

```diff
@@ -67,14 +67,14 @@ describe('ListItem', () => {
     })
   })
 
-  it('should not call on press callback if scrolling', async () => {
+  it('should not call on press callback if scrolling or swiping', async () => {
```

Comment:
I think two separate tests would make sense here, one that scrolling (only change y position) and one that swiping(only change x position) does not trigger on click. On the other hand the swipe to delete functionality is removed in #270 anyway. But I think just copying and adjusting would be easy and won't hurt.

Reply:
hm ok i overseen the fact that it gonna be removed. it we don't make any hotfix release. this pr is maybe not needed at all. So i won't put any more effort in tests to be honest